localcmd/local.go

```
@@ -56,16 +80,25 @@ func (p csvPutter) Put(ctx context.Context, ms []processor.MetricSample) error {
 			return err
 		}
 	}
+	defer p.writer.Flush()
 	return nil
 }
 
 func Run(args Args) (err error) {
 	logger := zap.NewNop()
-	csvp := csvPutter{
-		writer: *csv.NewWriter(os.Stdout),
+
+	var putter processor.MetricPutter
+	switch args.Format {
+	case FormatCSV:
+		putter = NewCSVPutter().Put
+	case FormatJSON:
+		putter = NewJSONPutter().Put
+	default:
+		logger.Error("unknown format")
```

Probably want to return an error here, since we're not actually going to do anything.

We'll probably get an error / panic later in the program because putter is nil anyway.